CODEOWNERS

```diff
@@ -1202,4 +1202,4 @@ build.json @home-assistant/supervisor
 /homeassistant/components/demo/weather.py @fabaff
 
 # Remove codeowners from files
-/homeassistant/components/*/translations/
\ No newline at end of file
+/homeassistant/components/*/translations/
```

Comment:
Hassfest did this during pre-commit... So it seems that another PR messed with this file? :)

Reply:
This is fine